bytes.go

```diff
@@ -71,7 +71,12 @@ func (x String) String() string {
 // Generate a random string. This method is implemented for use in quick tests.
 // See https://golang.org/pkg/testing/quick/#Generator for more information.
 func (String) Generate(r *rand.Rand, size int) reflect.Value {
-	v, _ := quick.Value(reflect.TypeOf(""), r)
+	v := reflect.New(reflect.TypeOf("")).Elem()
+	codePoints := make([]rune, size)
+	for i := 0; i < size; i++ {
+		codePoints[i] = rune(rand.Intn(0x10ffff))
+	}
+	v.SetString(string(codePoints))
 	return reflect.ValueOf(NewString(v.String()))
```

Comment:
Why not something simpler:
```suggestion
	runes := make([]rune, size)
	for i := 0; i < size; i++ {
		runes[i] = rune(rand.Intn(0x10ffff))
	}
	return reflect.ValueOf(String(string(runes)))
```